config/routes.rb

```diff
@@ -172,6 +173,10 @@
           # See the above :admin_settings routes, outside of :communities resource
           get :settings,       to: redirect("/admin/settings")
           put :update_settings # PUT request, no redirect
+          get "getting_started_guide(/*all)", to: redirect {|p, req|
+            sub_path = p[:all] == "" ? "" : "/#{p[:all]}"
+            "#{p[:locale]}/admin/getting_started_guide#{sub_path}"
+          }
```

Comment:
Repeating is possible alternative, but do we need this at all? 
This thing has been live for less than 1 week and we could probably survive without this. (or just redirection to guide root ```get "getting_started_guide(/*all)", to: redirect("/admin/getting_started_guide")```